src/util/uint.rs

```diff
@@ -516,13 +516,13 @@ impl Uint256 {
     #[inline]
     pub fn increment(&mut self) {
```

Comment:
It would be nice if we could move this into the macro generation, that way we can add a fuzz case for it for u128 and get fuzz coverage.

Reply:
I agree.

Reply:
Err, can you go ahead and move it there in this PR (can be a separate commit). That would require swapping the if branches for a loop, but that should be easy.

Reply:
Will do.